package.json

```
@@ -1,11 +1,23 @@
 {
   "name": "node-emoji",
-  "version": "1.10.0",
-  "description": "simple emoji support for node.js projects",
+  "version": "2.0.0",
+  "description": "Simple emoji support for Node.js.",
+  "repository": "https://github.com/omnidan/node-emoji.git",
   "author": "Daniel Bugl <daniel.bugl@touchlay.com>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/omnidan/node-emoji.git"
+  "license": "MIT",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "lint": "standard",
+    "test": "standard && ava",
```

```suggestion
    "test": "ava",
```

Also, out of curiosity: any reason for `ava`? Nothing against it; I just haven't seen many projects using it.

---

I think back then I had some issues with `jest`, which was a loooong time ago though. I think now `jest` would be the more standard and stable solution for this 🃏